specification/marketplacecatalog/data-plane/Products/preview/2023-01-01-preview/products.json

```
@@ -0,0 +1,1826 @@
+          },
+          {
+            "type": "string",
+            "name": "language",
+            "in": "query",
+            "description": "Language to search, ISO 639-1 two-letter code, possible values - 'en,cs,de,es,fr,hu,it,ja,ko,nl,pl,pt-br,pt-pt,ru,sv,tr,zh-hans,zh-hant'. Default is \"en\"",
+            "default": "en"
+          },
+          {
+            "type": "boolean",
+            "name": "includeStopSoldPlans",
+            "in": "query",
+            "description": "Denotes whether to include stop sold plans",
+            "default": false
+          },
+          {
+            "type": "array",
+            "name": "hideKeys",
+            "in": "query",
+            "description": "Add hide key to retrieve preview items. Hidekeys are provided in 'multi' format. Example: HideKey[]=first",
+            "collectionFormat": "multi",
+            "items": {
+              "type": "string"
+            }
+          },
+          {
+            "type": "string",
+            "name": "X-API-Key",
+            "in": "header",
+            "required": true,
+            "description": "Use Api key provided by Marketplace Catalog Team as a parameter provided in the header"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns a single public or preview product",
+            "schema": {
+              "$ref": "#/definitions/ProductDetails"
+            }
+          },
+          "default": {
+            "description": " Error Responses: \n\n * 400 - In case of a misconfigured request.\n\n * 401 - In case of a missing api key header\n\n * 404 - In case product not found.\n\n * 500 - In case of internal error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code"
+              }
+            }
+          }
+        }
+      }
+    },
+    "/products": {
+      "get": {
+        "tags": [
+          "ListingPublicProducts"
+        ],
+        "summary": "Public Products API provides an unauthenticated endpoint for consuming commercial Marketplace public products. In addition, it enables filtering on selected product properties, search, and getting starting price information per product. Each API request must include an API key allocated to you (see X-API-Key description).Send an email to MKPL_Platform_API_DL@microsoft.com including the following details to get the API Key:\n- Customer name\n- Service/Product name\n- AD Registered App ID (if available)\n- Focal point email\n- Use case/scenario\n- Expected traffic volume, including peak requests per second (daily)",
+        "x-ms-examples": {
+          "Product_Get": {
+            "$ref": "./examples/ListingPublicProducts_Get.json"
+          }
+        },
+        "operationId": "PublicProducts_Get",
```

The verb in the operationId for a GET that returns a collection should be "list" rather than "get".
```suggestion
        "operationId": "PublicProducts_List",
```

---

Fixed